go.mod

```diff
@@ -20,8 +20,8 @@ require (
 	google.golang.org/grpc v1.39.1 // indirect
 	gopkg.in/gcfg.v1 v1.2.0
 	gopkg.in/warnings.v0 v0.1.1 // indirect
-	k8s.io/api v0.22.0
-	k8s.io/apimachinery v0.22.0
+	k8s.io/api v0.22.1
+	k8s.io/apimachinery v0.22.1
```

Comment:
can we bump all of the kube libs to the same version?

Reply:
Sure, I'll try v0.22.1